scripts/check-pkg-for-release.js

```diff
@@ -162,6 +176,11 @@ function checkDependencyAddedInMDCPackage() {
 }
 
 function checkPkgDependencyAddedInMDCPackage() {
+  const name = getPkgName();
+  if (CSS_WHITELIST.filter((x) => JS_WHITELIST.includes(x)).includes(name)) {
```

Comment:
This is hard to read. What's it checking? Is there a cleaner way to write this, perhaps moving one part of it to a constant?

Reply:
I'll rewrite it to `foo.includes() && bar.includes()` be clearer - just set intersection